src/test/java/org/sagebionetworks/web/unitclient/widget/refresh/EntityRefreshAlertTest.java

```diff
@@ -37,49 +36,49 @@
 	SynapseJSNIUtils mockSynapseJSNIUtils;
 	@Mock
 	Etag mockEtag;
-	RefreshAlert refreshAlert;
+	EntityRefreshAlert entityRefreshAlert;
 
 	@Before
 	public void before() {
 		MockitoAnnotations.initMocks(this);
-		refreshAlert = new RefreshAlert(mockView, mockJsClient, mockGWTWrapper, mockGlobalApplicationState, mockSynapseJSNIUtils);
+		entityRefreshAlert = new EntityRefreshAlert(mockView, mockJsClient, mockGWTWrapper, mockGlobalApplicationState, mockSynapseJSNIUtils);
 		when(mockView.isAttached()).thenReturn(true);
 		AsyncMockStubber.callSuccessWith(mockEtag).when(mockJsClient).getEtag(anyString(), any(ObjectType.class), any(AsyncCallback.class));
```

Comment:
These tests pass??? If true, that's disheartening.  All jsClient.getEtag() checks (and mock responses) should be changed to jsClient.getEntity() checks (and mock responses).  Looks like a verification that mockView.setVisible(true) is missing (detection that etag has changed).